src/views/ipv/page/confirm-your-details.njk

```diff
@@ -123,7 +123,7 @@
         name: "detailsCorrect",
         fieldset: {
             legend: {
-            text: 'pages.confirmDetails.content.formTitle' | translate,
+            text: 'pages.confirmDetails.content.formTitle' | translate | safe,
```

Comment:
(I assume you've checked that including html into `text` works as expected!)

Reply:
<img width="963" alt="Screenshot 2024-06-17 at 17 40 18" src="https://github.com/govuk-one-login/ipv-core-front/assets/7995998/eacc3c6c-d52d-4ede-8ff6-c587dfbeaf55">